crates/humanode-runtime/src/lib.rs

```diff
@@ -810,6 +810,14 @@ impl pallet_erc20_support::Config for Runtime {
     type Metadata = EvmBalancesErc20Metadata;
 }
 
+frame_support::parameter_types! {
+    pub PrecompilesAddresses: Vec<H160> = vec![frontier_precompiles::hash(2050)];
```

Comment:
Do we only want `2050` (which is what? I suggest we declare a more meaningful way of referencing precompiles, for instance a `mod` with a bunch of `pub const QWE: ... = 2050;`-ish items at the `frontier_precompiles.rs`).

Reply:
I was thinking that `2050` is the only now to stored as code. Should we add more?

Precompiles constants have been added to simplify references - https://github.com/humanode-network/humanode/pull/777/commits/42d81213e82ccbd63220788e604d6a1851fac7b4.

Reply:
I suppose the standard precompiles don't need this, but anything that pretends to be a contract (emulates contract ABI and calls) and does not have a Solidity built-in syntax sugar (like `ecrecover`, et al) the dummy code would be helpful.

Reply:
https://github.com/humanode-network/humanode/pull/777/commits/ec1e1b0528b6ee3263f783e075ca6c40a7c6835c